src/keys/slip10.rs

```diff
@@ -485,6 +394,13 @@ impl KeyImpl {
     }
 }
 
+impl<K: hazmat::Derivable> TryFrom<&[u8; 65]> for Extended<K> {
+    type Error = crate::Error;
+    fn try_from(ext_bytes: &[u8; 65]) -> crate::Result<Self> {
+        Self::try_from_extended_bytes(ext_bytes)
+    }
+}
+
 #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
 pub struct Segment(u32);
```

Comment:
Can you add a getter/deref for the u32 here?
https://github.com/iotaledger/crypto.rs/issues/192

Reply:
What would be preferable solution?
1. Get rid of Segment type and use u32 directly.
2. Make inner field public.
3. Add accessor `fn into_u32(&self) -> u32;`.
4. `impl Deref/DerefMut for Segment`.
5. `impl From<u32> for Segment`, `impl From<Segment> for u32`.

Reply:
If there's no specific reason to define the type, then remove it. Otherwise, I would just make the inner field pub.